spark/v3.2/spark/src/test/java/org/apache/iceberg/spark/source/TestMetadataTablesWithPartitionEvolution.java

```diff
@@ -92,17 +105,11 @@
                 "cache-enabled", "false" // Spark will delete tables using v1, leaving the cache out of sync
             ),
             AVRO,
-            formatVersion()
+            1
```

Comment:
Shall we test v2 for Avro as well?

Reply:
Yea was trying to not increase the test time (seems it was original concern), but added it.